assets/js/components/setup/SetupUsingProxyWithSignIn.js

```diff
@@ -95,6 +101,22 @@ export default function SetupUsingProxyWithSignIn() {
 		};
 	} );
 
+	const dashboardURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getAdminURL( 'googlesitekit-dashboard' )
+	);
+
+	const canViewSharedDashboard = useSelect( ( select ) =>
+		select( CORE_USER ).hasCapability( PERMISSION_VIEW_SHARED_DASHBOARD )
+	);
+
+	const { dismissItem } = useDispatch( CORE_USER );
+
+	const goToSharedDashboard = useCallback( () => {
+		dismissItem( SHARED_DASHBOARD_SPLASH_ITEM_KEY );
+
+		global.location = dashboardURL;
```

Comment:
We should probably use `navigateTo` here instead of directly assigning to `global.location`...